apps/condo/domains/common/hooks/useExportTaskUIInterface.tsx

```diff
@@ -80,20 +82,37 @@ export const useExportTaskUIInterface = <T extends ExportTaskTypes> ({
                 const exportedRecordsCount = get(taskRecord, 'exportedRecordsCount')
 
                 return taskStatus === TASK_COMPLETED_STATUS
-                    ? ExportTaskProgressDescriptionCompleted.replace('{url}', get(taskRecord, 'file.publicUrl'))
-                    : !totalRecordsCount || !exportedRecordsCount
+                    ? (
+                        <>
+                            <Typography.Text>{ExportTaskProgressDescriptionCompleted}</Typography.Text>
+                            <br/>
+                            <Typography.Link href={get(taskRecord, 'file.publicUrl')}>
```

Comment:
Should `undefined` be default value? Sometimes it can cause some issues:
https://github.com/open-condo-software/condo/pull/3556

(You can use `#` as default or render element conditionally)